endpoints/tracking/index.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-param-reassign */
+
+import request from 'request'
+import cheerio from 'cheerio'
+import app from '../../server'
+
+app.get('/tracking/:trackingNumber', (req, res) => {
+  const url = 'http://www.postur.is/' +
+    'einstaklingar/senda-pakka-innanlands/finna-sendingu/' +
+    `?TrackingNumber=${req.params.trackingNumber}&Language=IS`
+
+  request.get(url, (err, response, body) => {
+    const $ = cheerio.load(body)
+
+    const history = $('table').eq(1).find('tr').map((index, row) => {
+      return {
+        date: $(row).children().eq(0).text()
+          .replace(/\s/g, ''),
+        action: $(row).children().eq(1).text()
+          .replace(/\s/g, ''),
+      }
+    })
+      .get()
+      .slice(1)
+      .reduce((sum, curr) => {
+        sum[curr.date] = curr.action
+        return sum
+      }, {})
+
+    if (Object.keys(history).length === 0) {
+      return res.status(404).json({})
+    }
+
+    return res.json({
```

Comment:
Cache it?

Reply:
I'm wondering how long tho. Presumably people would want to know the status of their package right now? Maybe a cache of a minute would help if we are getting bombarded with requests tho?

Reply:
I'd say any caching is good. 1 minute sounds good. We can always tweak it later if we feel the need.